shards/modules/core/time.hpp

```diff
@@ -31,6 +30,26 @@ struct DeltaTimer {
     return delta.count();
   }
 };
+
+struct DoOnce {
```

Comment:
Where do we use this?

Reply:
I just used it in a place to log a warning message but only every 2 seconds at most
(similar to the old timer warning, but this time in fbl)